src/wallet/rpcwallet.cpp

```diff
@@ -2632,6 +2632,7 @@ static void LockWallet(CWallet* pWallet)
     LOCK(pWallet->cs_wallet);
     pWallet->nRelockTime = 0;
     pWallet->Lock();
+    pWallet->LockStealthAddresses();
```

Comment:
This is for locking stealth address when the timer runs out (after unlocking for a time amount)